amazon-kinesis-client/src/test/java/software/amazon/kinesis/utils/MockObjectHelper.java

```diff
@@ -0,0 +1,31 @@
+package software.amazon.kinesis.utils;
+
+import software.amazon.awssdk.regions.Region;
+import software.amazon.awssdk.services.kinesis.KinesisAsyncClient;
+import software.amazon.awssdk.services.kinesis.KinesisServiceClientConfiguration;
+
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+public final class MockObjectHelper {
+
+    public static KinesisAsyncClient createKinesisClient() {
+        return createKinesisClient(Region.US_EAST_1);
+    }
+
+    /**
+     * @param isRegionDummy a boolean to determine whether to use a null value for the Kinesis client's region.
+     * @return
+     */
+    public static KinesisAsyncClient createKinesisClient(boolean isRegionDummy) {
```

Comment:
DRY+KISS: this method is redundant with the calls in the ternary. This doesn't simplify anything, and introduces more code we must maintain.